src/components/Identity/index.js

```diff
@@ -124,13 +127,8 @@ const createIdentity = ({ config, logger, cookieJar }) => {
       getEcid() {
         return optIn.whenOptedIn().then(getEcid);
       },
-      // TODO: Discuss renaming of CustomerIds to UserIds
       setCustomerIds(options) {
-        return optIn
-          .whenOptedIn()
```

Comment:
Where did the `whenOptedIn()` go? It looks like we're no longer waiting for opt-in before reading the CUSTOMER_ID_HASH cookie within `customerIds.sync`.